plugin/pkg/scheduler/schedulercache/cache.go

```diff
@@ -60,6 +60,8 @@ type podState struct {
 	pod *v1.Pod
 	// Used by assumedPod to determinate expiration.
 	deadline *time.Time
+	// Used to block cache from expiring assumedPod if binding still runs
+	waitsBinding chan struct{}
```

Comment:
I'm slightly against introducing a channel (which is not for free) for the purpose of handling bool value. You only use this channel to check if it is open or not. Please use bool for this purpose.